autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/SectionSelect/SectionInfo.tsx

```diff
@@ -130,7 +130,7 @@ const SectionInfo: React.FC<SectionInfoProps> = ({
         </Tooltip>
       );
     }
-    return <span>{MeetingType[mtg.meetingType]}</span>;
+    return <>{MeetingType[mtg.meetingType]}</>;
```

Comment:
I would drop the Fragment and return `JSX.Element | string`

Reply:
oh shit I had no idea you could do that